app/utils/TypografiWrapper.tsx

```diff
@@ -28,12 +28,20 @@ export const TypografiWrapper: React.FC<Props> = ({
           {children}
         </Heading>
       );
+    case TypografiTyper.HeadingH2:
```

Comment:
Jeg tenker at både Banner og overskriften til steget bør være `<h1>` 😊 https://www.uutilsynet.no/veiledning/tekst-og-struktur/226#innhold_som_ikke_skal_kodes_som_overskrifter

Tenker også det fint å få ryddet opp litt kode/refaktorere dersom man toucher den koden. Ser at steg 1 bruker `HeadingH2. Så hva er `StegHeadingH1` og `StegHeadingH2` og hvor brukes det? 

Bør `HeadingH2` egentlig være `StegHeadingH1` og at den nåværende StegHeadingH1 hete noe annet?

Reply:
Hva med `StegHeadingSmallH1` ? @daphneleebeek 
Trur tanken var at man skulle ha H1 men size large og ikkje extra large.

Reply:
Ja, det synes jeg er mer beskrivende. Men synes fortsatt forskjellen ikke er så tydelig. I følge stegindikatoren så er jo forsiden ikke et steg. Så ved første øyekast ville jeg tenkt at her kan stegene ha to forskjellig typer overskrifter, en normal/default og en liten `StegHeadingH1` og `StegHeadingSmallH1`. I realiteten så brukes StegHeadingH1 kun til overskrift på forsiden såvidt jeg kan se? og de faktiske stegene kan bare ha én type overskrift? 😊

Reply:
I såfall ville jeg nok gått for noe som `HeadingH1` på forsiden og `StegHeadingH1` for stegene.

Reply:
Se fra commit f01fb09
Trur det er riktig nå 🤩